content/docs/ref/pr.md

```diff
@@ -3,6 +3,59 @@
 Commit specified files to a new branch and create a pull request. If sending a
 report afterwards, consider using `cml send-comment --pr --update`.
 
+ⓘ Pull requests created with `cml pr` **won't** trigger a new CI/CD run
+under any circumstances.
+
+ⓘ You can specify the files to track using any syntax supported by [Git pathspec](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefpathspecapathspec).
```

Comment:
```suggestion
ⓘ You can specify the files to track using any syntax supported by
[Git pathspec](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefpathspecapathspec).
Note that globs may have to be quoted to avoid shell expansion (e.g. `"*.py"`
not `*.py`).
```

Reply:
> _Note that globs may have to be quoted to avoid shell expansion_

Do we _really_ want to encourage replacing native `extglob` with out internal globbing?

Reply:
idk. Sing Git itself supports both and we link to its docs, maybe we don't need to draw extra attention to it. Maybe discard this suggestion? :thinking:

Reply:
```suggestion
ⓘ Files to commit can be specified using any syntax supported by
[Git pathspec](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefpathspecapathspec).
```